config.go

```diff
@@ -321,6 +321,8 @@ type config struct {
 
 	MaxChannelFeeAllocation float64 `long:"max-channel-fee-allocation" description:"The maximum percentage of total funds that can be allocated to a channel's commitment fee. This only applies for the initiator of the channel. Valid values are within [0.1, 1]."`
 
+	EnableUpfrontShutdown bool `long:"enable-upfront-shutdown" description:"If true, option upfront shutdown script will be enabled. If peers that we open channels with support this feature, we will set the script to which cooperative closes should be paid out to upon channel opening, and allow peers to do the same. This offers the partial protection of a channel peer diconnecting if cooperative close is attempted with a different script, but does not protect against force closes."`
```

Comment:
not used?

Reply:
Called in `getUpfrontShutdown` to check whether we should set new addresses?